Using-the-Compiler-API.md

```diff
@@ -140,6 +140,8 @@ fileNames.forEach(fileName => {
 
 In this example, we did not need to create a type checker because all we wanted to do was traverse each `SourceFile`.
 
+All possible ```ts.SyntaxKind``` can be found under enum [here](https://github.com/Microsoft/TypeScript/blob/master/src/compiler/types.ts#L80).
```

Comment:
i would rather we have a link to the `lib\typescript.d.ts` since this is what users get when they build against TS

Reply:
@mhegazy done. changed link to typescript.d.ts